code/modules/admin/admin_fax_panel.dm

```diff
@@ -108,9 +107,8 @@
 			if(params["fromWho"])
 				sending_fax_name = params["fromWho"]
 			
-			if(!save_reminder)
-				to_chat(usr, span_info("Do not forget to close and open the preview after saving."))
-				save_reminder = TRUE
+			if(fax_paper.ui_status(usr)) // Closes paper UI, for.. user handle re-open.
```

Comment:
Do you want to UPDATE or do you want to CLOSE? Also `ui_status` is a proc that is meant to be pure

Reply:
> Do you want to UPDATE or do you want to CLOSE? Also `ui_status` is a proc that is meant to be pure

i want to do something. i cant update, but i can close(or update, but use timers).

Reply:
@Mothblocks i think... i find!

https://user-images.githubusercontent.com/78199449/201224268-7208d5f0-fff9-41ab-8e72-f5cf211617db.mp4